pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -1330,17 +1329,17 @@ func doNetworkInstanceActivate(ctx *zedrouterContext,
 	return err
 }
 
-func getSwitchIPv4Addr(ctx *zedrouterContext, bridgeIndex int) (string, bool) {
+func getSwitchIPv4Addr(ctx *zedrouterContext, bridgeIndex int) (net.IP, bool) {
 	addrs, _, err := ctx.networkMonitor.GetInterfaceAddrs(bridgeIndex)
 	if err == nil {
 		for _, addr := range addrs {
 			if addr.IP.IsLinkLocalUnicast() {
 				continue
 			}
-			return addr.String(), true
```

Comment:
Is this from where we got the string with /24 at the end?

Reply:
And does it explain how that string with /24 propagate to the acl.go code and get fed to iptables?

Reply:
Yes. Actually the problematic iptables rule in this case is created from `createServer4`.

Reply:
Hmm, then I don't understand why the other NAT rules were missing on one test box. Let me give it a spin.